Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -359,4 +382,46 @@ protected function getRequirementsFromConfig($fileName, $config)
 
         return $requirements;
     }
+
+    /**
+     * Parses the '--SETTINGS--' block of a '.test' file and determines settings.
+     *
+     * @param string $fileName
+     * @param string $config
+     *
+     * @return array
+     */
+    protected function getSettingsFromConfig($fileName, $config)
```

Comment:
why `FromConfig`? I think it is misleading because there is a `--CONFIG--` block.